packages/shared/lib/sdk/sync.ts

```diff
@@ -312,18 +312,18 @@ interface MetadataChangeResponse {
 }
 
 interface Connection {
-    id?: number;
-    created_at: Date;
-    updated_at: Date;
+    id: number;
     provider_config_key: string;
     connection_id: string;
     connection_config: Record<string, string>;
-    environment_id: number;
-    metadata?: Metadata | null;
-    credentials_iv?: string | null;
-    credentials_tag?: string | null;
+    created_at: string;
+    updated_at: string;
+    last_fetched_at: string;
+    metadata: Record<string, unknown> | null;
+    provider: string;
+    errors: { type: string; log_id: string }[];
+    end_user: ApiEndUser | null;
     credentials: AuthCredentials;
-    end_user_id: number | null;
 }
```

Comment:
I haven't removed the whole typings but it would break everybody that does not install nango in a package.json